controllers/wishlist.controller.js

```diff
@@ -0,0 +1,86 @@
+const User = require("../models/user.model");
+const Wishlist = require("../models/wishlist.model");
+
+const getWishlists = async (req, res) => {
+  const wishlists = await Wishlist.find({});
+  res.json({ success: true, wishlists });
+};
+
+const findUserWishlist = async (req, res, next, userId) => {
+  try {
+    let user = await User.findOne({ _id: userId });
+    if (!user) {
+      res
+        .status(404)
```

Comment:
you can define the status values as constants & pass those in the .status() function. Here's what I mean:
declaration :`const PAGE_NOT_FOUND=404`
usage: `.status(PAGE_NOT_FOUND)`